Add origami-version GitHub Action

JakeChampion opened the pull request.

origamiserviceuser commented:
o-visual-effects bundle size difference from 3.0.0 to 1a2879ea9efc9be5eb38c6d0a1c6193cd3d58344
No significant bundle size differences found.

JakeChampion committed:
Add origami-version GitHub Action

When this is merged, we can have automated releases by using the release-type labels "major"/"minor"/"patch" on a pull-request before merging the pull-request.

origamiserviceuser commented:
o-visual-effects bundle size difference from 3.0.0 to 0bdd0df45373bb3e6f7bf1ad4a13f1deaa2fc706
No significant bundle size differences found.

JakeChampion commented:
This helps achieve the OKR outlined here https://github.com/Financial-Times/origami/issues/39
